content/en/docs/concepts/storage/volumes.md

```diff
@@ -1404,72 +867,493 @@ plugin path on each node and in some cases the control plane nodes as well.
 Pods interact with FlexVolume drivers through the `flexvolume` in-tree volume plugin.
 For more details, see the [FlexVolume](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-storage/flexvolume.md) examples.
 
-## Mount propagation
-
-Mount propagation allows for sharing volumes mounted by a container to
-other containers in the same pod, or even to other pods on the same node.
+### persistentVolumeClaim {#persistentvolumeclaim}
 
-Mount propagation of a volume is controlled by the `mountPropagation` field
-in `Container.volumeMounts`. Its values are:
+A `persistentVolumeClaim` volume is used to mount a
```

Comment:
I think the most imprtant aspect of a PVC is that the lifecycle is independent of a pod

Reply:
I added a note calling this out, thanks